src/interface.jl

```diff
@@ -0,0 +1,32 @@
+export stable_intersection
+
+"""
+    stable_intersection(S::MixedSupport, M::RealisableMatroid)
+
+Computes the stable intersection given the data of a mixed support `S` and a realisable matroid `M`.
+"""
+function stable_intersection(S::MixedSupport, M::RealisableMatroid)
+    Δ, σ = starting_data(S, M)
+    T = tracker(Δ, S, [σ], path=:coefficient_wise)
+
+    return TropicalHomotopies.stable_intersection(T)
+end
+
+"""
+    compute_mixed_support(v::Vector{TropicalPolynomial})
+
+Computes the mixed support associated to a vector of polynomials `v`.
+"""
+function compute_mixed_support(v::Vector)
```

Comment:
Could we rename this to `construct_mixed_support` or, maybe better, just have a constructor `mixed_support` in `mixed_support.jl`?  Using `compute` is unintuitive, because nothing is computed here.